doorman/application.py

```diff
@@ -63,7 +65,11 @@ def register_loggers(app):
     import logging
     from logging.handlers import WatchedFileHandler
 
-    handler = WatchedFileHandler(app.config['DOORMAN_LOGGING_FILENAME'])
+    log_fname = app.config['DOORMAN_LOGGING_FILENAME']
+    if log_fname == 'sys.stdout':
```

Comment:
Would rather we have some code here like:

```
if isinstance(log_fname, file):
    handler = logging.StreamHandler(log_fname)
elif isinstance(log_fname, basestring):
    handler = WatchedFileHandler(log_fname)
```

That way, we can use either `sys.stdout`, `sys.stderr`, or other stream type.